client/landing/gutenboarding/stores/onboard/types.ts

```diff
@@ -38,3 +38,5 @@ export interface Vertical {
 	preview_styles_url: string;
 	synonyms: string[];
 }
+
+export type DomainName = string;
```

Comment:
Shouldn't define here IMO, since we're already defining it in the domain suggestions store, and the definitions could collide. I'd suggest just changing the import paths for the `DomainName` type in this store's reducer and actions file (which I'd messed up in the first place :roll_eyes: )

Reply:
(Per a video call, let's remove this from this PR and tackle separately.)